src/drivers/dw/dma.c

```diff
@@ -273,6 +273,9 @@ static int dw_dma_start(struct dma_chan_data *channel)
 
 	irq_local_disable(flags);
 
+	if (channel->status == COMP_STATE_ACTIVE)
```

Comment:
can we have an comment here stating pause is unreliable and DAI must do the pausing.

Reply:
Wont this break the state transition rules? How can you distinguigh between a previously started ACTIVE state vs a previously released ACTIVE state?